### ISSUE TITLE
Write functions to generate paths for the shapes we need to draw

### ISSUE THREAD

**Issue by JackBuck** [JackBuck/Robo-Plot#20]
What constitutes a path is to be defined by issue #18 .

The shapes we need to draw are
- Straight line
- Circles
- Circular Arcs (the rules document is vague since svg arcs include elliptical arcs, but we will start with circular arcs)

These should also be chainable.

---
**EDIT:** This issue has now become responsible for SVGPath curves as well.

**Comment by JackBuck** [comment]
On hold while we investigate converting SVG paths to a series of points.

**Comment by JackBuck** [comment]
SVG curves appear to work (superficially!) but there is a bug in the debug_movement.py module which needs fixing before we can be 100% sure!

**Comment by JackBuck** [comment]
I just want to test this on an svg file with multiple curves now.

**Comment by JackBuck** [comment]
Hopefully just one bug left where the disjoint path making up the bubble man has not been drawn disjoint...

**Comment by JackBuck** [comment]
Ok, ready for review!!

**Comment by Hsenrab** [comment]
So I added tests note that not all passing as they throw errors. Once these are fixed the outputs for the tests need to generated and committed. @JackBuck

**Comment by JackBuck** [comment]
Merged in #78
